README.md

```diff
@@ -450,26 +450,38 @@ an [observable](https://github.com/dominictarr/obv) of the current log sequence.
 is always a positive integer that usually increases, except in the exceptional circumstance
 that the log is deleted or corrupted.
 
-### db.addBoxer(box)
+### db.addBoxer(boxer)
 
-add a box, which will be added to the list of boxers which will try to
+add a `boxer`, which will be added to the list of boxers which will try to
 automatically box (encrypt) the message `content` if the appropriate
 `content.recps` is provided.
 
-`box` is a function of signature `box(content, recps) => (String|null)`
-which is expected to either box the the message content and return a ciphertext String, or return null if it unable to.
+`box` is a function of signature `box(content, recps, cb)` which is expected either:
+- successfully box, calling back with a `ciphertext` String - `cb(null, ciphertext)`
+- communicate that it can't box this - `cb(null, null)`
+- communicate that it thought it could box this but hit an error - `cb(err)`
 
 ### db.addUnboxer({key: unboxKey, value: unboxValue})
 
 add an unboxer object, any encrypted message is passed to the unboxer object to
 test if it can be unboxed (decrypted)
 
 where
-- `unboxKey(ciphertext) => msgKey` is a function which tries to extract the message key from the encrypted content (`ciphertext`)
-- `unboxValue(ciphertext, msgKey) => plaintext` is a function which takes a message key and uses it to try to extract the message content from the `ciphertext`
+- `unboxKey(msg.value.content, msg.value, cb)`
+  - is a function which tries to extract the message key from the encrypted content (`ciphertext`).
+  - is expected to callback `cb(err, msgKey)` which `msgKey` is the key for the message
+- `unboxValue(msg.value.content, msgKey, msg.value, cb)`
```

Comment:
I'm sure there's a good reason why, but I don't understand:  could this just be `unboxValue(msg, cb)`? Seems like we've destructured `msg` into its parts and are passing `content` twice (in `msg.value.content` and `msg.value`).

Reply:
I'm going to copy this comment into a comment I made about this.
TL;DR I agree